examples/browser/test/workspace/search-in-workspace.ts

```diff
@@ -0,0 +1,128 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { TopPanel } from '../top-panel/top-panel';
+import { LeftPanel } from '../left-panel/left-panel';
+
+export class SearchInWorkspace {
+
+    /** accessors */
+    private readonly _searchField = '#search-input-field';
+    private readonly _includeField = '#include-glob-field';
+    private readonly _excludeField = '#exclude-glob-field';
+
+    private readonly _searchInfo = '.search-info';
+    private readonly _options = '//*[@id="search-in-workspace"]/div/div[1]/div[1]/div[3]/div[1]/span';
```

Comment:
Can we add id to span to look up it easier and make tests less dependent on dom?